partner_phone_search/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,3 @@
+* Serpent Consulting Services Pvt. Ltd. <support@serpentcs.com>
+* Nikul Chaudhary <nikul.chaudhary.serpentcs@gmail.com>
+* Nikul Chaudhary <nikulchaudhary2112@gmail.com>
```

Comment:
Is this a different Nikul Chaudhary?

Reply:
@tarteo One is Official and Another one is personal

Reply:
I think you should put only one or put both in the same line.